safehttp/plugins/csp/csp.go

```diff
@@ -233,17 +241,34 @@ func (it Interceptor) Before(w safehttp.ResponseWriter, r *safehttp.IncomingRequ
 	for _, p := range it.ReportOnly {
 		reportCSPs = append(reportCSPs, p.Serialize(nonce))
 	}
+	// X-Frame-Options works only in enforce mode.
+	var XFO = []string{getXFO(&it.Enforce)}
 
 	h := w.Header()
 	setCSP := h.Claim("Content-Security-Policy")
 	setCSPReportOnly := h.Claim("Content-Security-Policy-Report-Only")
+	setXFO := h.Claim("X-Frame-Options")
 
 	setCSP(CSPs)
 	setCSPReportOnly(reportCSPs)
+	setXFO(XFO)
```

Comment:
Won't this always set "SAMEORIGIN"?

Shouldn't we *not* set XFO if a framing policy is not set?